src/main/webapp/confirmation.jsp

```diff
@@ -0,0 +1,38 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Confirmation Page</title>
+</head>
+<body>
+<jsp:include page="userInformations.jsp" />
+
+<h4>Subscription</h4>
+<c:set var="user" value="${sessionScope.user}"/>
+<c:set var="subscription" value="${sessionScope.subscription}"/>
+<c:set var="servicePackage" value="${subscription.getServicePackage()}"/>
+<c:set var="validityPeriod" value="${subscription.getValidityPeriod()}"/>
+<c:set var="startDate" value="${subscription.getStartDate()}"/>
+<c:set var="optionalProducts" value="${subscription.getProducts()}"/>
+<c:set var="totalPrice" value="${validityPeriod.getMonthlyFee() * validityPeriod.getNumberOfMonths()}"/>
```

Comment:
the total price needs to be calculated in the back end - this is a business logic functionality and it shouldn't be in the frontend; also the value needs to be stored on the order; the cleanest way is to create a service that does the calculation of the total

Reply:
Yeah you're right, If you have a better solution you can modify the code without any problem

Reply:
I will refactor it